docs/CxIAST-Integration.md

```diff
@@ -0,0 +1,82 @@
+* [Suggested Flow](#suggestedFlow)
+* [Configuration](#configuration)
+* [Filters](#filters)
+* [Thresholds](#thresholds)
+* [Bug Trackers](#bugTrackers)
+
+## <a name="suggestedFlow">Suggested Flow</a>
+CxIAST can be integrated within a CI/CD pipeline using CxFlow.  
+A CxIAST scan starts automatically when an application with an attached CxIAST agent is being tested.  
+CxIAST scans can be stopped in two ways:
+1. Manually stopped via the CxIAST manager.
+2. By terminating the running application.
+
+Onces tests are done, CxFlow is used to stop the running CxIAST scan.  
+Then, CxFlow will collect the results from CxIAST, analyze them, and open tasks/tickets in the configured bug tracker.  
+To make sure the correct scan is stopped by CxFlow, CxIAST scan tags are used:  
```

Comment:
Onces tests are completed, CxFlow is used to stop the running CxIAST scan.  CxFlow will then collect the results from CxIAST, analyze them, and open tasks/tickets in the configured Bug Tracker [put link to Bug Tracker here].  To make sure the correct scan is stopped by CxFlow, CxIAST scan tags are used